deployment/Dockerfile_BE

```diff
@@ -8,13 +8,15 @@ COPY .mvn /build/.mvn
 
 RUN ./mvnw -B dependency:go-offline
 
-ENV SPRING_DATABASE_URL=jdbc:postgresql://192.168.1.57:5432/auctionapp?useSSL=false
-ENV SPRING_DATABASE_USERNAME=postgres
-ENV SPRING_DATABASE_PASSWORD=auction
-ENV aws_access_key=AKIA4KCKUJQPBPW3EJPE
-ENV aws_secret_key=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
+ENV SPRING_DATASOURCE_URL=jdbc:postgresql://192.168.1.57:5432/auctionapp?useSSL=false
+ENV SPRING_DATASOURCE_USERNAME=postgres
+ENV SPRING_DATASOURCE_PASSWORD=auction
+ARG aws_access_key
+ARG aws_secret_key
 ENV APP_JWTSECRET=d178899b7beeb99ec2ebdad4636c7e21a2364494414013f78dd5a00725b77f14
```

Comment:
You should also save `APP_JWTSECRET` as secret. We do not want to share this value in public.